### PR TITLE
FAQ on 2FA.

### LINKED ISSUE #997: FAQ: Can we add 2fa to wallets?
A discussion which has happened multiple times on Matrix. [Most recent example](https://matrix.to/#/!wSdymYrEpBhsWlDJuk:decred.org/$15706037277677WgAls:decred.org?via=decred.org&via=matrix.org&via=zettaport.com)

We could capture this as an FAQ on https://docs.decred.org/faq/wallets-and-seeds/

### PR DESCRIPTION
**Pull request by jholdstock**
Closes #997

### CODE REVIEW

File: docs/faq/wallets-and-seeds.md [hunk 1]
```diff
@@ -80,6 +80,39 @@ You should backup your `wallet.db` file (preferably in a thumb drive stored in a
...
 
+#### 9. Could wallet security be improved using 2-Factor Authentication (2FA)?
+
+2-Factor Authentication (2FA) would provide no extra security to Decrediton or
+dcrwallet.
+
+2FA is an authentication technique in which a computer user is granted access
+only after successfully presenting two or more pieces of evidence (or factors)
+to an authentication mechanism.
+These are typically "something you know" such as a password, and "something you
```
Comment (s-ben) [inline docs/faq/wallets-and-seeds.md]: Add more commas for flow? I.e.

> These are typically “something you know”, such as a password, and “something you possess”, such as a smartphone.

Comment (jholdstock) [reply docs/faq/wallets-and-seeds.md]: agreed

**Comment by degeri** [comment]
lgtm

**Comment by RichardRed0x** [comment]
lgtm

**Pull request merged**